styles/styles.css

```diff
@@ -12,61 +12,33 @@
 
 :root {
   /* colors */
-  --link-color: #035fe6;
-  --link-hover-color: #136ff6;
+  --link-color: #f00;
+  --link-hover-color: #1d1d1b;
   --background-color: white;
```

Comment:
Can you change this to  `--c-primary-white: #fff`.  Use it in line 179 (for button text color)

Reply:
Chagned